extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -541,11 +541,11 @@ public static function getHostByDbName( $dbName ) {
 	 * @return string
 	 */
 	public static function getHostById( $cityId ) {
-		global $wgDevelEnvironment;
+		global $wgWikiaEnvironment;
 
 		$hostName = \WikiFactory::getVarValueByName( 'wgServer', $cityId );
 
-		if ( !empty( $wgDevelEnvironment ) ) {
+		if ( $wgWikiaEnvironment != WIKIA_ENV_PROD ) {
 			$hostName = \WikiFactory::getLocalEnvURL( $hostName );
 		}
```

Comment:
As you stated in the PR desc, the same logic is [here](https://github.com/Wikia/app/blob/dev/includes/wikia/GlobalTitle.php#L715-L722). Why not simply call it here?